### PR TITLE
Improve test coverage for module Bio.Pathway

### PR DESCRIPTION
**Pull request by MaxGreil**
This pull request addresses issue #...

#1288 

- Improves test coverage for module `Bio.Pathway`.
- Additionally fixed 2 errors in `Bio.Pathway.Rep.Graph` in functions `__repr__` and `edges` and 1 error in `Bio.Pathway.Rep.MultiGraph` in function `edges

I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
Why do you need to add a unique check in ``__repr__``? i.e. ``unique(list(self._adjacency_list[key]))``

Here ``self._adjacency_list`` is a dict, keyed on source nodes, values are sets of destination nodes as sets. i.e. ``self._adjacency_list[key]`` should be a set, so cannot have any duplicates by construction.

Also, that aside, I would avoid using ``numpy.unique`` here as that makes the whole module depend on numpy unnecessarily.

**Comment by MaxGreil** [comment]
@peterjc sorry for the long delay. I used `numpy.unique` in `Graph.py` to get the same functionality for function `__repr__` as in` MultiGraph.py`. However, if you think that including `numpy.unique` I will try to achieve the same functionality witht it.

**Comment by peterjc** [comment]
I don't understand why you want to use ``numpy.unique`` in ``Graph.py`` to match ``MultiGraph.py`` since that does not use it either.

Why do you think there is an error in the current ``Graph``'s ``__repr__`` function?

**Comment by MaxGreil** [comment]
@peterjc you were right, I changed the code and removed `numpy.unique` from `Graph.py`.

**Comment by MaxGreil** [comment]
I now added `sorted` not only for `MultiGraph.py` but also for` Graph.py`.

**Comment by MaxGreil** [comment]
@peterjc I now added `sorted` not only for `MultiGraph.py` but also for` Graph.py`.

File: Bio/Pathway/Rep/MultiGraph.py [hunk 1]
```diff
@@ -79,11 +80,11 @@ def edges(self, label):
         """Return a list of all the edges with this label."""
         if label not in self._label_map:
             raise ValueError("Unknown label: " + str(label))
-        return sorted(self._label_map[label])
+        return sorted(list(self._label_map[label]))
```
Comment (peterjc) [inline Bio/Pathway/Rep/MultiGraph.py]: I don't think we need ``list(...)`` here, do we?

File: Bio/Pathway/Rep/MultiGraph.py [hunk 2]
```diff
@@ -79,11 +80,11 @@ def edges(self, label):
...
 
     def labels(self):
         """Return a list of all the edge labels in this graph."""
-        return list(self._label_map.keys())
+        return sorted(list(self._label_map.keys()))
```
Comment (peterjc) [inline Bio/Pathway/Rep/MultiGraph.py]: Again, wouldn't ``return sorted(self._label_map.keys())`` be enough here?

**Comment by MaxGreil** [comment]
@peterjc I removed the list()-function. The Travis CI tests are still working. However, in the appveyor tests was an error at the end: 
```
codecov --file Tests/coverage.xml -X pycov -X gcov
'codecov' is not recognized as an internal or external command, operable program or batch file.
Command exited with code 1
```

**Comment by peterjc** [comment]
The AppVeyor thing is strange - just above this in the logs you can see ``conda install -c conda-forge codecov`` worked.

According to https://anaconda.org/conda-forge/codecov the package was updated yesterday with ``noarch/codecov-2.0.15-py_0.tar.bz2`` replacing older python-specific packages for codecov 2.0.9

From the timing, this is most likely an error in that conda package, which I will report now.

**Comment by peterjc** [comment]
Merged, thank you.

I've opened https://github.com/conda-forge/codecov-feedstock/issues/3 and will try something like this as a workaround:

```
conda install -c conda-forge codecov=2.0.9
```

Would you like to be thanked by name in the ``NEWS.rst`` file for what will be Biopython 1.73?

**Comment by MaxGreil** [comment]
I'd like to be thanked in the `News.rst`. Thank you.

**Comment by peterjc** [comment]
I'll add you to the ``NEWS.rst`` file - can you suggest how to improve the wording about this in the pull request template?
https://github.com/biopython/biopython/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by MaxGreil** [comment]
Thank you very much. I am sorry but I don't quite get your question. What do you mean by improve the wording about this (I suggest adding to the `NEWS.rst` file) in the pull request template?

**Comment by peterjc** [comment]
I added you to the ``NEWS.rst`` file as part of ac97d06d38ce605f85f983640a6e4b0f9656ce2c

I would like people to add themselves as part of the pull request. Often people do not, so I wonder how we can change the wording in the pull request template to encourage this:
https://github.com/biopython/biopython/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Pull request merged**